pom.xml

```diff
@@ -360,12 +408,13 @@
 						<phase>process-sources</phase>
 						<configuration>
 							<target>
-								<mkdir dir="${basedir}/src/main/webapp/less/bootstrap"/>
+								<mkdir dir="${project.build.directory}/staged-less/bootstrap"/>
+								<move tofile="${project.build.directory}/staged-less/" file="${project.basedir}/src/main/webapp/less/" />
 								<move 
-									tofile="${basedir}/src/main/webapp/less/bootstrap" 
-									file="${basedir}/src/main/webapp/less/webjars/META-INF/resources/webjars/bootstrap/${version.bootstrap}/less/" 
+									tofile="${project.build.directory}/staged-less/bootstrap" 
+									file="${project.build.directory}/less-webjars/META-INF/resources/webjars/bootstrap/${version.bootstrap}/less/" 
 								/>
-								<delete dir="${basedir}/src/main/webapp/less/webjars"/>
```

Comment:
the source should be good to go like this.  The compile-on-save configuration in your ide needs to be changed a little to reflect the new structure.

These are the lines netbeans added to my nb-configuration.xml
```
<org-netbeans-modules-css-prep.less_2e_configured>true</org-netbeans-modules-css-prep.less_2e_configured>
        <org-netbeans-modules-css-prep.sass_2e_compiler_2e_options/>
        <org-netbeans-modules-css-prep.less_2e_mappings>/less:${web.root}/../../../../target/sparrow-ui/css</org-netbeans-modules-css-prep.less_2e_mappings>
        <org-netbeans-modules-css-prep.less_2e_enabled>true</org-netbeans-modules-css-prep.less_2e_enabled>
        <org-netbeans-modules-css-prep.sass_2e_mappings>/scss:/css</org-netbeans-modules-css-prep.sass_2e_mappings>
        <org-netbeans-modules-css-prep.sass_2e_enabled>false</org-netbeans-modules-css-prep.sass_2e_enabled>
        <org-netbeans-modules-css-prep.less_2e_compiler_2e_options>--include-path=${web.root}/../../../target/staged-less</org-netbeans-modules-css-prep.less_2e_compiler_2e_options>
```

Main thing here to keep a note of is the repointing of the output directory to inside the target, rather than in src/main/webapp.  Other part is the compiler option `include-path` this will allow us to keep the import directive the same, but the compiler will look for the file elsewhere.